ui-lib/build.gradle.kts

```diff
@@ -122,6 +131,9 @@ dependencies {
     implementation(libs.zcash.bip39)
     implementation(libs.zxing)
 
+    api("com.flexa:core:1.0.1")
+    api("com.flexa:spend:1.0.1")
```

Comment:
Can we use the same dependency approach as we do elsewhere, injecting them via `settings.gradle`?

Reply:
fixed